### PR TITLE
Proposed updates to target size enhanced and target size minimum to a…

### PR DESCRIPTION
**Pull request by rachaelbradley**
…ddress #3075

### CODE REVIEW

File: understanding/21/target-size-enhanced.html [hunk 1]
```diff
@@ -19,6 +19,7 @@ <h2>Intent</h2>
 			  <li>the control is positioned where it will be difficult to reach, or is near the edge of the screen;</li>
 			  <li>the control is part of a sequential task.</li>
 	  </ul>
+	  <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size as the target. It is recommended that when the visible indicator of the target is notably smaller than the target, that the visible indicator be placed at the center of the target.</p>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:22]: As per my comments in the issue, I'd prefer we not get into this space. But if we're going to add some content, I don't see why it is only germane to small indicators. We'd want big indicators to be centered on the target too.
```suggestion
	  <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size or shape as the target. It is recommended that when the visible indicator of the target differs from the target, that the visible indicator be centered on the target.</p>
```

Comment (patrickhlauke) [inline understanding/21/target-size-enhanced.html:22]: I believe we always expand "SC" to "Success Criterion" (here and in the other file change)

File: understanding/22/target-size-minimum.html [hunk 2]
```diff
@@ -101,6 +101,8 @@ <h3>Size requirement</h3>
           
           <p>While the Success Criterion primarily helps touch users by providing target sizing to prevent accidental triggering of adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
 
+          <p>This SC does not require that the visible indicator of a target, such as a button or icon, be the same size as the target. It is recommended that when the visible indicator of the target is notably smaller than the target, that the visible indicator be placed at the center of the target.</p>
```
Comment (mbgower) [inline understanding/22/target-size-minimum.html:104]:
```suggestion
          <p>This Success Criterion does not require that the visible indicator of a target, such as a button or icon, be the same size or shape as the target. It is recommended that when the visible indicator of the target differs from the target, the visible indicator be centered on the target.</p>
```

**Comment by patrickhlauke** [comment]
As the original issue that this came from has been closed, and there was strong opposition in that issue, wondering if this PR should also be closed

**Comment by mbgower** [comment]
Given the closure of the issue by the issue opener, we are not proceeding with this PR.

**Comment by gundulaniemann** [comment]
There are reasonable invisible targets, like
whitespace inside a card, where the card also contains visible targets like action buttons.

**Pull request closed without merging**